tmt-app/src/Components/Breadcrumb.jsx

```diff
@@ -26,9 +27,10 @@ function Breadcrumb({ previousPage }) {
         },
       }}
     >
-      {breadcrumbs.map((breadcrumb) => {
+      {breadcrumbs.map((breadcrumb, idx) => {
```

Comment:
Can we avoid the abbreviation? i think `idx` is the first time i've seen this shorthand for index lol